cfe_internal/update/update_policy.cf

```diff
@@ -46,6 +49,10 @@ bundle agent cfe_internal_update_policy
 
   methods:
 
+    # if def.policy_servers is available then select the first available policy server (for next agent run)
+    have_def_policy_servers::
+      "Select first available policy server"
```

Comment:
mmm if we aren't using this class more than once, we could simply put the condition on an if =>  right here.